tests/unit/base/trained_model_serialization_test.cc.py

```diff
@@ -15,7 +15,7 @@
     'CMultitaskL12LogisticRegression', 'CMultitaskLeastSquaresRegression',
     'CMultitaskTraceLogisticRegression', 'CMultitaskClusteredLogisticRegression',
     'CLatentSVM', 'CLatentSOSVM', 'CDomainAdaptationSVMLinear',
-    'CLinearLatentMachine',
+    'CLinearLatentMachine','CPerceptron',
```

Comment:
why this?

Reply:
because is was not being initialized earlier in the same way... should be fine when we overload train_machine and use members instead of args for features